client/src/components/AuthModal/AuthForm/index.tsx

```diff
@@ -29,65 +40,75 @@ export const AuthForm = (props: AuthFormProps) => {
         },
         onSubmit: (values: FormValues, { setSubmitting }): void => {
             setSubmitting(false)
+            isLogin
+                ? userStore.login(values)
+                : userStore.registration(values)
```

Comment:
it's just a nitpick, do however you want - ternary operator is used primarily for assigning values(ex. const a = 1 ? b : c), not for conditional invocation.